### PR TITLE
ZTE-SH-CN-define-environment-variable-container

### PR DESCRIPTION
**Pull request by YuxiJin-tobeyjin**
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> For 1.8 Features: set Milestone to `1.8` and Base Branch to `release-1.8`
> ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>
> NOTE: Please check the “Allow edits from maintainers” box (see image below) to
> [allow reviewers to fix problems](https://help.github.com/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork/) on your patch and speed up the review process.

Allow edits from maintainers checkbox
https://github.com/kubernetes/kubernetes.github.io/blob/master/docs/tasks/inject-data-application/define-environment-variable-container.md

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/5466)

### CODE REVIEW

**Comment by YuxiJin-tobeyjin** [comment]
CC @zhangxiaoyu-zidif

**Comment by zhangxiaoyu-zidif** [comment]
@Bradamant3 @chenopis 
LGTM
It seems that the lack of directory inject-data-application results in *deploy failed*

File: cn/docs/tasks/inject-data-application/define-environment-variable-container.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+{% capture steps %}
+
+## 为容器设置一个环境变量
+
+创建Pod时，可以为其下的容器设置环境变量。通过配置文件的`env`或者`envFrom` 字段来设置环境变量。
+
+本示例中，将创建一个只包含单个容器的Pod。Pod的配置文件中设置环境变量的名称为`DEMO_GREETING`，
+其值为`"Hello from the environment"`。下面是Pod的配置文件内容:
+
+{% include code.html language="yaml" file="envars.yaml" ghlink="/docs/tasks/inject-data-application/envars.yaml" %}
```
Comment (chenopis) [inline cn/docs/tasks/inject-data-application/define-environment-variable-container.md]: You might want to copy the **envars.yaml** file into the `/cn/docs/tasks/inject-data-application/` directory and update the path here. That should fix the Netlify preview.

**Comment by YuxiJin-tobeyjin** [comment]
@chenopis

**Comment by zhangxiaoyu-zidif** [comment]
@chenopis 
could you approve this PR?

**Comment by Bradamant3** [comment]
Approving to clean up the queue -- it looks as though the changes that @chenopis requested were made.

**Pull request merged**